keg_auth/testing.py

```diff
@@ -19,65 +20,72 @@ class AuthTests(object):
     """
         These tests are designed so they can can be imported into an application's tests
         and ran to ensure customization of KegAuth hasn't broken basic functionality.
-
-        TODO: the messages we test for need to be configurable on the class in case the app
-        customizes then.  Ditto some of the redirect logic.
     """
-    login_url = '/login'
     protected_url = '/secret1'
```

Comment:
In order to make this generic add a class attribute which allows subclasses to add permissions to the user who is trying to hit the `proected_url` and add those to the user during `testing_create`.